app/main/views/drafts.py

```diff
@@ -347,6 +348,44 @@ def complete_draft_service(draft_id):
     return jsonify(services=draft.serialize()), 200
 
 
+@main.route('/draft-services/<int:draft_id>/update-status', methods=['POST'])
```

Comment:
This is quite different from the service status update URL, while doing a similar thing. I think we should look at them and decide on a more consistent endpoint URL.